### DISCUSSION TITLE
Schedule:test promts default scroll value

### DISCUSSION THREAD

**Discussion by ChrisSantiago82** [laravel/framework#48692] [Ideas]
The new prompts are super nice. But I'm having a little issue with the "Schedule:test" command. It shows 5 options, and then it starts to scroll. I always have tons of options, and if I need to select the correct one out of 40, and it's only showing 5, it's not easy.

Am I the only one with this issue? Maybe the default value could be 10 or something like that.

What do you think?

**Comment by henzeb** [comment]
Here too. @jessarcher has created a version with search. Implemented it for the vendor:publish command. Maybe this can be done here as well?

**Comment by ChrisSantiago82** [comment]
Either add the search, or put the scroll to 10 or something like that.
